### PR TITLE
[miio] add alert on miio channels changing

### PR DESCRIPTION
**Pull request by marcelrv**
related to  ao https://github.com/openhab/openhab-addons/pull/9756 and https://github.com/openhab/openhab-addons/pull/9384

Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 1]
```diff
@@ -46,6 +46,9 @@ DEFAULT;$OPENHAB_USERDATA/etc/org.ops4j.pax.logging.cfg
 DEFAULT;$OPENHAB_USERDATA/etc/log4j2.xml
 DEFAULT;$OPENHAB_USERDATA/etc/org.ops4j.pax.logging.cfg
 
+[3.1.0]
+ALERT;Xiaomi MiIo binding: With 3.1 release many channels have been converted from Number to a quanityType number. e.g. temperature channels have been changed from Number to Number:Temperature. You need to update your item definition to the new definitions. Please see the binding readme for the item details for your device.
```
Comment (kaikreuzer) [inline distributions/openhab/src/main/resources/bin/update.lst:50]:
```suggestion
ALERT;Xiaomi MiIo Binding: Many channels have been converted from Number to a Number with dimension, e.g. temperature channels have been changed from Number to Number:Temperature. Please see the binding documentation for details.
```

**Review by kaikreuzer — CHANGES_REQUESTED** [review]
Question: Is this really a breaking change?
It should always be possible to link a dimensionless item to such a channel, so ideally this should be backward compatible.

**Comment by marcelrv** [comment]
> Question: Is this really a breaking change?
> It should always be possible to link a dimensionless item to such a channel, so ideally this should be backward compatible.

To be honest I'm still not 100% clear indeed if it is breaking or not. I recall I had some issues with rules after the change overs.
Nevertheless, I'm very fine with the rewording you did.

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 2]
```diff
@@ -46,6 +46,9 @@ DEFAULT;$OPENHAB_USERDATA/etc/org.ops4j.pax.logging.cfg
 DEFAULT;$OPENHAB_USERDATA/etc/log4j2.xml
 DEFAULT;$OPENHAB_USERDATA/etc/org.ops4j.pax.logging.cfg
 
+[3.1.0]
```
Comment (kaikreuzer) [inline distributions/openhab/src/main/resources/bin/update.lst:49]: I'd think this sections needs to be moved to https://github.com/openhab/openhab-distro/pull/1247/files#diff-8be360a52adcae6cc237c7d0e4658b2d50028192596d6489f64ee61e4d43f4aaR40, i.e. under `[[MSG]]` and not under `[[PRE]]`.

**Review by kaikreuzer — CHANGES_REQUESTED** [review]

**Review by kaikreuzer — APPROVED** [review]
Thanks!

**Pull request merged**